tests/test_landings.py

```diff
@@ -29,9 +29,13 @@ def db(app):
         _db.drop_all()
 
 
-def test_landing_revision_saves_data_in_db(db, client, phabfactory):
+def test_landing_revision_saves_data_in_db(
+    db, client, phabfactory, transfactory
+):
     phabfactory.user()
     phabfactory.revision()
+    transfactory.land()
```

Comment:
transfactory does not depend on revision - it's just returning the request_id under which it saved the landing request, I might make the number to be set with this command. The status of the Landing request is set using pingback, which is tested in another test.